actions/filter-files/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Filter files please'
+description: 'Filter the list of changed files'
+inputs:
+  changed-files:
+    description: 'jsonified list of changed files from setup'
+    required: true
+  files:
+    description: 'comma-separated list of files to check for'
+    required: false
+  extensions:
+    description: 'comma-separated list of extensions to check for'
+    required: false
+outputs:
+  filtered:
+    description: 'The jsonified list of files that match'
+    value: ${{ steps.result.outputs.result }}
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/github-script@v6
+      id: result
+      with:
+        script: |
+          const extensionsRaw = "${{ inputs.extensions }}";
```

Comment:
I wonder if we should have a policy (in a future change) of breaking out inline scripts to be imported from another file? That way we can lint and test them, for example: https://github.com/actions/github-script#run-a-separate-file

Reply:
on interesting, I didn't know that was an option! Sounds like a good policy.

Reply:
Always been possible, I believe!